src/db/migrations/39-add-kota.ts

```diff
@@ -0,0 +1,51 @@
+import { Knex } from 'knex';
+
+import { ArtistIdExtractorTypes, ArtistNameExtractorTypes, IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { MetaFactory, MetaFactoryTypeName } from '../../types/metaFactory';
+import { NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { Table } from '../db';
+import { addMetaFactory, addPlatform, updateViews } from '../migration-helpers';
+
+const KOTA_PLATFORM: MusicPlatform = {
+  id: 'kota',
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'KOTA',
+}
+
+const KOTA: MetaFactory = {
+  id: '2ZvRTpStD4gQ9WPoXuiA4MtFpiZze9K9fPfFtCTnzdGo',
+  platformId: KOTA_PLATFORM.id,
+  contractType: MetaFactoryTypeName.candyMachine,
+  standard: NFTStandard.METAPLEX,
+  autoApprove: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        name: 'Kids of the Apocalypse'
+      },
+      extractor: {
+        id: IdExtractorTypes.USE_TITLE_EXTRACTOR,
+        title: TitleExtractorTypes.METADATA_NAME_WITHOUT_TRAILING_INFO,
+        artistId: ArtistIdExtractorTypes.USE_ARTIST_ID_OVERRIDE,
+        artistName: ArtistNameExtractorTypes.USE_ARTIST_NAME_OVERRIDE,
+      }
+    }
+  }
+};
+
+export const up = async (knex: Knex) => {
+
+  await knex.schema.alterTable(Table.metaFactories, table => {
+    table.jsonb('typeMetadata')
+  })
+
+  await addPlatform(knex, KOTA_PLATFORM);
+  await addMetaFactory(knex, KOTA);
```

Comment:
Can we switch to a CRDT instead of doing direct platform + metafactory injection? Hoping to rely entirely on seeds for new platforms and contracts and building out the parts that don't work as expected.